azbb-scripts/abbt-cluster-manual.json

```diff
@@ -28,7 +28,7 @@
               },
               "nics": [
                 {
-                  "isPublic": false,
+                  "isPublic": true,

```

Comment:
There's no need to make this public. All access is done thru the Jumpbox. No additional PIPs required this way.